app/services/xcode_manager_service.rb

```diff
@@ -0,0 +1,170 @@
+require_relative "../shared/logging_module"
+require_relative "./services"
+
+require "xcode/install"
+
+module FastlaneCI
+  # Manages Xcode installations
+  class XcodeManagerService
+    include FastlaneCI::Logging
+
+    # Keeps an a hash, the key being the Gem::Version of the Xcode version we're installing
+    # and the value being the % of the download progress
+    attr_accessor :installations_in_progress
+
+    def initialize
+      if ENV["XCODE_INSTALL_USER"].to_s.length == 0
+        raise "No `XCODE_INSTALL_USER` ENV variable provided, please provide one when launching up the fastlane.ci"
+      end
+
+      self.installations_in_progress = {}
```

Comment:
This is a nitpick, but could we change this to be `@installations_in_progress = {}` to be consistent with how we're setting variables with `attr_accessor`s in other clases?

Reply:
Yep